src/services/books/saga.ts

```diff
@@ -14,7 +14,10 @@ const DEFAULT_BOOKS_ID_CHUNK_SIZE = 60;
 
 function* fetchBooks(bIds: string[]) {
   const data = yield call(pRetry, () => requestBooks(bIds), { retries: 2 });
+  const descData = yield call(pRetry, () => requestBooksDesc(bIds), { retries: 2 });
```

Comment:
```suggestion
  const [data, descData] = yield all([
    call(pRetry, () => requestBooks(bIds), { retries: 2 }),
    call(pRetry, () => requestBooksDesc(bIds), { retries: 2 }),
  ]);
```

진짜 아주아주 중요합니다 이거 이렇게 안 하면 동시에 진행이 안 돼요

Reply:
아 call 구문을 따로 all 로 합치지 않으면 각각 블로킹 순차로 도는군요! 👍